src/container/index.tsx

```diff
@@ -24,6 +24,7 @@ export default function Container({
       disableContentPaddings={disableContentPaddings}
       {...externalProps}
       {...baseComponentProps}
+      __hiddenContent={!props.children}
```

Comment:
Does this mean that all internal use of container remain unfixed? Why is that needed?

This fix should be applied to internal instances too

Reply:
Basically, [this line](https://github.com/cloudscape-design/components/blob/main/src/container/internal.tsx#L104) should become

```
[styles['with-hidden-content']]: !children || __hiddenContent
```

Reply:
Hi Boris, thanks for checking it. I was not sure we should not use container without content internally at all, It should be either ExpandableSection or Header? But I agree it makes sense the fix should be for all cases.

Reply:
fix in https://github.com/cloudscape-design/components/pull/669